crates/biome_markdown_parser/src/lexer/mod.rs

```diff
@@ -181,10 +179,31 @@ impl<'src> MarkdownLexer<'src> {
         match dispatched {
             WHS => self.consume_newline_or_whitespace(),
             MUL | MIN | IDT => self.consume_thematic_break_literal(),
+            HAS => self.consume_header(),
```

Comment:
What is `HAS` in this context? Should it be `HASH`?

Reply:
https://github.com/biomejs/biome/blob/main/crates/biome_unicode_table/src/bytes.rs#L26